GVFS/GVFS.FunctionalTests/Tests/GitCommands/AddStageTests.cs

```diff
@@ -32,12 +32,30 @@ public void StageBasicTest()
         }
 
         [TestCase, Order(3)]
+        [Category(Categories.Mac.M2)]
+        public void AddAndStageHardLinksTest()
+        {
+            if (!this.FileSystem.SupportsHardlinkCreation)
+            {
+                return;
+            }
+
+            this.CreateHardLink("Readme.md", "ReadmeLink.md");

```

Comment:
Nit: in the `IPlatformFileSystem` interface up above, I saw that the link name was first and the target name was second. I think it would help us keep these straight if we keep a consistent ordering through all of our own code.

Reply:
:+1: I'll clean this up